test/form/samples/pure-class-field/_expected.js

```diff
@@ -0,0 +1,6 @@
+class main {
+	static k = /*#__PURE__*/ V();
+	static /*#__PURE__*/ ignored;
```

Comment:
Does this sample line has special meaning in rollup? It seems not a pure mark for function call.
Is it the new feature of rollup?
Or just as a test, to make sure here is no bug, without any other use?

Reply:
The latter, a test to cover a branch in the logic even if it makes no sense to do so in real code.